.github/workflows/application-health-checker.yml

```diff
@@ -27,3 +29,14 @@ jobs:
 
       - name: Build Project # Build with npm
         run: npm run build
+
+      - uses: sonarsource/sonarqube-scan-action@v3
+        env:
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+          SONAR_HOST_URL: ${{ secrets.SONAR_HOST_URL }}
+      # If you wish to fail your job when the Quality Gate is red, uncomment the
+      # following lines. This would typically be used to fail a deployment.
+      # - uses: sonarsource/sonarqube-quality-gate-action@master
+      #   timeout-minutes: 5
+      #   env:
+      #     SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
```

Comment:
@sanshigo345 @plnknr burayı kullanmamız gerekmiyor mu? Quality Gate!e göre hareket etmesini sağlamamız gerekiyor diye düşünüyorum

Reply:
Evet sonarqube integration yapıldı bu kısım back-end tarafında yoktu o yüzden bilgim yoktu araştırdım anladığım kadarıyla kalite kriterleri karşılanmadığında pipeline da devam etmemesi için gereken bir ayar, tekrardan branch oluşturup bu kısmı değiştirip pull request oluşturuyorum @sanshigo345

Reply:
@plnknr tekrardan branch oluşturup, bu kısmı değiştirip, pull request oluşturmadan önce bu kısmın olup olmaması gerektiğinden emin olabilir miyiz? ayrıca kalite kriterlerimiz nedir, karar verildi mi, belgelendi mi? kalite kriterleri belli olmadan kalite kriterlerini karşılamadığında pipeline devam etmesin demek doğru olmayabilir.